cypress/e2e/eda/Credentials/credentials-crud.cy.ts

```diff
@@ -36,12 +36,12 @@ describe('EDA Credentials- Create, Edit, Delete', () => {
     cy.clickButton(/^Create credential$/);
     cy.typeInputByLabel(/^Name$/, name);
     cy.typeInputByLabel(/^Description$/, 'This is a GitHub Credential.');
-    cy.selectDropdownOptionByLabel(/^Type$/, 'GitHub Personal Access Token');
+    cy.selectDropdownOptionByLabel(/^Type$/, 'personal access token');
```

Comment:
@jamestalton did we mean to drop the `GitHub` at the beginning of this string?

Reply:
looking - thanks for catching that